lint-staged.config.js

```diff
@@ -1,11 +1,10 @@
 module.exports = {
```

Comment:
IMO use `.json` config file instead of `.js`

Reply:
because of yarn type:check command we need to use function model here